src/v2i-hub/CARMAStreetsPlugin/manifest.json

```diff
@@ -1,48 +1,64 @@
-		},
-		{
-			"key":"transmitMobilityOperationTopic",
-			"default":"v2xhub_mobility_operation_in",
-			"description":"Apache Kafka topic plugin will transmit message to."
-		},
-		{
-			"key":"transmitMobilityPathTopic",
-			"default":"v2xhub_mobility_path_in",
-			"description":"Apache Kafka topic plugin will transmit message to."
-		},
-		{
-			"key":"KafkaBrokerIp",
-			"default":"127.0.0.1",
-			"description":"IP of Apache Kafka broker."
-		},
-		{
-			"key":"KafkaBrokerPort",
-			"default":"9092",
-			"description":"Port of Apache Kafka broker."
-		},
-		{
-			"key":"MobilityOperationStrategies",
-			"default":"carma/port_drayage,TruckInspection",
-			"description":"A comma separated list of strategies of MobilityOperation messages to send to CARMA Streets"
-		}
-		
-	]
-}
+    "name": "CARMAStreetsPlugin",
+    "description": "Plugin to communicate with CARMA Streets via Apache Kafka",
+    "version": "@PROJECT_VERSION@",
+    "exeLocation": "/bin/CARMAStreetsPlugin",
+    "coreIpAddr": "127.0.0.1",
+    "corePort": 24601,
+    "messageTypes": [{
+        "type": "J2735",
+        "subtype": "TMSG03-P",
+        "description": "..."
+    }],
+    "configuration": [{
+            "key": "receiveTopic",
+            "default": "v2xhub_out",
+            "description": "Apache Kafka topic plugin will receive message from."
+        },
+        {
+            "key": "transmitMobilityOperationTopic",
+            "default": "v2xhub_mobility_operation_in",
+            "description": "Apache Kafka topic plugin will transmit message to."
+        },
+        {
+            "key": "transmitBSMTopic",
+            "default": "v2xhub_bsm_in",
+            "description": "Apache Kafka topic plugin will transmit message to."
+        },
+        {
+            "key": "runKafkaConsumer",
+            "default": "1",
+            "description": "indicator for consuming kafka messages."
+        },
+        {
+            "key": "subscribeToSchedulingPlanTopic",
+            "default": "v2xhub_scheduling_plan_sub",
+            "description": "Apache Kafka topic plugin will transmit message to."
+        },
+        {
+            "key": "transmitMobilityPathTopic",
+            "default": "v2xhub_mobility_path_in",
+            "description": "Apache Kafka topic plugin will transmit message to."
+        },
+        {
+            "key": "intersectionType",
+            "default": "Carma/stop_controlled_intersection",
+            "description": "The type of intersection"
+        },
+        {
+            "key": "KafkaBrokerIp",
+            "default": "127.0.0.1",
+            "description": "IP of Apache Kafka broker."
+        },
+        {
+            "key": "KafkaBrokerPort",
+            "default": "9092",
+            "description": "Port of Apache Kafka broker."
+        },
+        {
+            "key": "MobilityOperationStrategies",
+            "default": "carma/port_drayage,TruckInspection,Carma/stop_controlled_intersection",
```

Comment:
We should probably set the default value to just stop_controlled

Reply:
It is Carma/stop_controlled_intersection as we discussed.

Reply:
yeah but we should probably remove carma/port_drayage and TruckInspection right

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/271/commits/076713611a1da7b5e14e745c4ff8e8fe6ff96381